falcon/request.py

```diff
@@ -619,11 +623,11 @@ def date(self):
 
     @property
     def if_match(self):
-        return helpers.parse_etags(self.env.get('HTTP_IF_MATCH'))
```

Comment:
Nit-pick: Any reason we can't just store these in a private variable on first access instead of parsing every time?

Reply:
Well, I was debating whether to add the extra overhead of another variable slot and having to initialize it. I guess we could use the class variable shadowing trick?

Reply:
I think we could just try releasing as-is now, and revise in the future if needed.
I wouldn't take it for granted that this property is accessed many times, I could imagine one would just check this value only once to render the `Not Modified` or similar.

Reply:
Fixed